src/VegaWrapper2.js

```diff
@@ -116,6 +116,9 @@ class VegaWrapper2 {
         const host = urlObj.wiki ? urlObj.wiki : options.domain;
         const sanitizedHost = this.sanitizeHost(host);
 
+        // Path to api.php, e.g. "/w" for Wikimedia projects.
+        const scriptPath = options.scriptPath !== undefined ? options.scriptPath : '/w'
```

Comment:
I am not certain, but wouldn't this be simpler?  I might be a bit rusty on my javascript though.  Also, needs a semicolon.

```suggestion
        const scriptPath = options.scriptPath ?? '/w';
```

Reply:
`??` would also allow for `null` (which I wouldn't think would be a problem here), but is indeed otherwise equivalent; however, depending on your versions of JavaScript support, this "nullish coalescing operator" was added more recently in JavaScript. Still, is now pretty good: https://caniuse.com/mdn-javascript_operators_nullish_coalescing  .

Reply:
You could also do `options.scriptPath || '/w';` i think

Reply:
That would unfortunately choose the default when the script path was the empty string, which I imagine could be a problem here.

Reply:
I don't think an empty string is valid - it must start with a `/`, or else it won't work

Reply:
oh, oops, you are right.... I think the better path would be to introduce a full API endpoint url rather than a part of it.  E.g. `apiEndpoint = "/w/api.php"` ? This would allow the proper override, wouldn't it?

Reply:
I don't have any strong opinions about it myself, but I can see your point. Not sure if the submitter, @edwardspec , has any opinions about it...

Reply:
Doesn't matter to me. Feel free to change the syntax if needed.